Branch-SDK-TestBed/res/values/strings.xml

```diff
@@ -3,6 +3,6 @@
 
     <string name="app_name">Branch-SDK-TestBed</string>
     
-    <string name="bnc_app_key">5680621892404085</string>
+    <string name="bnc_app_key">5668720416392049</string>
```

Comment:
I couldn't find it in the dashboard, but I named 5668720416392049 Branch Testing App in case you want a dashboard to play with on prod